ingestion/src/metadata/ingestion/source/database/dbt.py

```diff
@@ -510,16 +510,18 @@ def create_dbt_query_lineage(
         logger.info(f"Processing DBT Query lineage for: {table_fqn}")
 
         try:
-            source_elements = table_fqn.split(".")
+            source_elements = fqn.split(table_fqn)
+            # remove service name from fqn to make it parseable in format db.schema.table
+            query_fqn = ".".join(source_elements[-3:])
             query = (
-                f"create table {table_fqn} as {data_model_link.datamodel.sql.__root__}"
+                f"create table {query_fqn} as {data_model_link.datamodel.sql.__root__}"
```

Comment:
service name should not in included while constructing this query otherwise it breaks [here](https://github.com/open-metadata/OpenMetadata/blob/1a00e81fe82c3b37627638e54800429dcce6d06d/ingestion/src/metadata/ingestion/lineage/sql_lineage.py#L131) with `too many values to extract` error